src/test/librbd/object_map/test_mock_InvalidateRequest.cc

```diff
@@ -61,6 +61,18 @@ TEST_F(TestMockObjectMapInvalidateRequest, UpdatesHeadOnDiskFlag) {
   C_SaferCond cond_ctx;
   AsyncRequest<> *request = new InvalidateRequest<>(*ictx, CEPH_NOSNAP, false, &cond_ctx);
 
+  bool rwl_enabled = ictx->config.template get_val<bool>("rbd_rwl_enabled");
```

Comment:
Nit: I don't see any changes to the `InvalidateRequest` state machine so where are these coming from?

Reply:
Removed the check.